draft-irtf-cfrg-frost.md

```diff
@@ -607,14 +607,14 @@ a distinguished Coordinator; see {{no-coordinator}} for more information.
 FROST produces signatures that are indistinguishable from those produced with a single
 participant using a signing key `s` with corresponding public key `PK`, where `s` is a Scalar
 value and `PK = G.ScalarBaseMult(s)`. As a threshold signing protocol, the group signing
-key `s` is secret-shared amongst each participant and used to produce signatures. In particular,
+key `s` is Shamir secret-shared amongst each of the `MAX_PARTICIPANTS` participant and used to produce signatures. In particular,
 FROST assumes each participant is configured with the following information:
 
 - An identifier, which is a Scalar value denoted `i` in the range `[1, MAX_PARTICIPANTS]`
   and MUST be distinct from the identifier of every other participant.
-- A signing key share `sk_i`, which is a Scalar value representing the i-th secret share
-  of the group signing key `s`. The public key corresponding to this signing key share
-  is `PK_i = G.ScalarBaseMult(sk_i)`.
+- A signing key `sk_i`, which is a Scalar value representing the i-th Shamir secret share
+  of the group signing key `s`. In particular, `sk_i` is the point `f(i)` on a secret polynomial `f`, where `s` is `f(0)`.
```

Comment:
In the spec, `sk_i` is just a Scalar, forming a point together with `i`. Not sure the best wording here, maybe just

```suggestion
  of the group signing key `s`. In particular, `sk_i` is the value `f(i)` on a secret polynomial `f`, where `s` is `f(0)`.
```

Also maybe we can expand what the polynomial is, e.g. "a secret polynomial with of degree MIN_PARTICIPANTS-1 with randomly generated coefficients, where `s` is `f(0)`.

Reply:
Using `value` is better.